packages/treasury/test/swingsetTests/governance/vat-zoe.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+
+// noinspection ES6PreferShortImport
+import { E } from '@agoric/eventual-send';
+import { makeZoeKit } from '@agoric/zoe';
+
+export function buildRootObject(vatPowers) {
+  return Far('root', {
+    buildZoe: vatAdminSvc => {
+      const shutdownZoeVat = vatPowers.exitVatWithFailure;
+      const { zoeService, feeMintAccess } = makeZoeKit(
+        vatAdminSvc,
+        shutdownZoeVat,
```

Comment:
There's nothing specific to treasury governance in this `vat-zoe.js`, is there? This looks like boilerplate as a consequence of metering changes.

IOU an issue to consolidate it in `zoe/tools` or some such.

Reply:
that would be nice.  Yes, it's boilerplate, used repeatedly in many swingset tests.